bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/UnzipUtility.java

```diff
@@ -0,0 +1,103 @@
+package com.adobe.acs.tools.csv_asset_importer.impl;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+/**
+ * This utility extracts files and directories of a standard zip file to a
+ * destination directory.
+ *
+ */
+public class UnzipUtility {
+	/**
+	 * Size of the buffer to read/write data
+	 */
+	private static final int BUFFER_SIZE = 4096;
+
+	/**
+	 * Extracts a zip file specified by the zipFilePath to a directory specified
+	 * by destDirectory (will be created if does not exists)
+	 * 
+	 * @param zipFilePath
+	 * @param destDirectory
+	 * @throws IOException
+	 */
+	public static void unzip(String zipFilePath, String destDirectory) throws IOException {
```

Comment:
AFAIK, this method is never used and is largely duplicative of the next method. Please remove it.